config/globals.js

```diff
@@ -44,7 +44,7 @@ const globals = {
 	REQUEST_TIMEOUT: process.env.REQUEST_TIMEOUT || 8000,
 	METRICS_PATH: process.env.METRICS_PATH || '/metrics',
 	MONGOOSE_CONNECTION_POOL_SIZE: parseInt(process.env.MONGOOSE_CONNECTION_POOL_SIZE || '10', 10),
-	STUDENT_TEAM_CREATE_DISABLED: process.env.STUDENT_TEAM_CREATE_DISABLED,
+	STUDENT_TEAM_CREATION: process.env.STUDENT_TEAM_CREATION,
```

Comment:
use the configuration instead, remove from globals and replace cases using globals with configuration above.

Reply:
Configurations are used everywhere. Should I remove the setting from globals anyway?

Reply:
yes, please.

Reply:
done